grails-app/services/au/org/ala/collectory/SitemapService.groovy

```diff
@@ -14,12 +14,16 @@
  */
 package au.org.ala.collectory
 
+import grails.gorm.transactions.Transactional
+import grails.util.Holders
 import org.springframework.scheduling.annotation.Scheduled
 
 import java.text.SimpleDateFormat
 
 class SitemapService {
 
+    static lazyInit = !Holders.getConfig().getProperty("sitemap.enabled", Boolean.TYPE, false)
```

Comment:
If sitemap is enabled the SitemapService should be loaded on application startup thereby enabling scheduled runs. Otherwise scheduled runs are enabled only after a manual run from admin UI.